odoo_sh/getting_started/branches.rst

```diff
@@ -0,0 +1,316 @@
+After that, they can be garbage collected automatically.
+
+.. _odoosh-gettingstarted-branches-mergingbranches:
+
+Merging your branches
+---------------------
+You can merge your branches easily by drag and dropping them on each other.
+
+.. image:: ./media/interface-branches-merge.png
+   :align: center
+
+When you want to test the changes of your development branches with the production data,
+you can either:
+
+* merge the development branch into your staging branch, by drag and dropping it onto the desired staging branch,
+* drag and dropping the development branch on the staging section title, to make it become a staging branch.
+
+When your latest changes are ready for production,
+you can drag and drop your staging branch onto your production branch
+to merge and deploy in production your newest revisions.
+
+If you are bold enough,
+you can merge your development branches into your production branch as well.
+It just means you skip the validation of your changes with the production data through a staging branch.
+
+You can merge your development branches into each other, and your staging branches into each other.
+
+Of course, you can also use :code:`git merge` directly to merge your branches.
+Odoo.sh will be notified when new revisions have been pushed in your branches.
+
+Merging a staging branch in the production branch only merges the source code: Any configuration changes you made in the
+staging databases are not passed to the production database.
+
+If you test configuration changes in staging branches, and want them to be applied in the production, you have to:
+* Either, write the configuration changes in XML data files
+overriding the default configuration or views in your branches,
+and then increase the version of your module in its manifest (*__manifest__.py*) to trigger the update of the module
+when you merge your staging branch in your production branch.
+This is the best practice for a better scalability of your developments as you will use the Git versioning features
+for all your configuration changes, and therefore have a traceability for your changes.
+* Either, pass them manually from your staging to your production database, by copy/pasting them.
+
+.. _odoosh-gettingstarted-branches-tabs:
+
+Tabs
+=============
+
+History
+-------
+An overview of your branch history:
+
+* The messages of the commits and their authors,
+* The various events linked to the platform, such as stage changes, database imports, backup restores.
+
+.. image:: ./media/interface-branches-history.png
+   :align: center
+
+For each event, a status is displayed in the above right corner.
+It can provide information about the ongoing operation on the database (installation, update, backup import, ...),
+or its result (tests feedback, successful backup import, ...).
+When an operation is successful, you also got
+the possibility to access the database thanks to the *connect* button.
```

Comment:
When an operation is successful, you can access the database thanks to the *connect* button.